_docs/latest/setup/forseti-on-gke.md

```diff
@@ -220,6 +220,18 @@ helm upgrade -i forseti forseti-security/forseti-security \
     --set-string serverConfigContents="$(gsutil cat gs://<BUCKET_NAME>/configs/forseti_conf_server.yaml | base64 -)" \
     --values=forseti-values.yaml
 ```
+
+### Deploying with config-validator on GKE
+
+The config-validator in Forseti on-GKE obtains policies from a policy-library in a Git repository via SSH.  The pre-requisites for this are as follows.
```

Comment:
Remove dash in `Forseti on-GKE` --> `Forseti on GKE`

Reply:
Ditto